atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/timestamp/InDbTimestampBoundStore.java

```diff
@@ -72,32 +62,39 @@ public static InDbTimestampBoundStore create(
             ConnectionManager connManager,
             TableReference timestampTable,
             String tablePrefixString) {
-        InDbTimestampBoundStore inDbTimestampBoundStore = new InDbTimestampBoundStore(
-                connManager,
-                timestampTable,
-                tablePrefixString);
+        return createWithStrategy(connManager, new LegacyPhysicalBoundStoreStrategy(timestampTable, tablePrefixString));
+    }
 
-        inDbTimestampBoundStore.init();
+    public static InDbTimestampBoundStore createForMultiSeries(
+            ConnectionManager connManager,
+            TableReference timestampTable,
+            String series) {
+        return createWithStrategy(connManager, new MultiSequencePhysicalBoundStoreStrategy(timestampTable, series));
+    }
```

Comment:
I see we're removing the option of providing tablePrefix for multi series, which should be fine. Maybe we should should use Client rather than String for type checking?

Reply:
Ah. We can't use Client, because that's a part of leader-election-api and this is in atlasdb-dbkvs. However, I agree that having a string wrapper type is a good suggestion, I'll add one.